From 5fc667c39fea018411131281417b58b01e71174c Mon Sep 17 00:00:00 2001
From: Zhao Chao <zhaochao1984@gmail.com>
Date: Thu, 27 Nov 2014 15:09:50 +0800
Subject: [PATCH] avoid host name collision.

Reapply 1f55418e60e667c9da04b8dd21e678fb634a8813.

This is a regression bug, introduced by commit
a294fdfc7a410aadb6fd1c72594989e2f3ef1a26.

Signed-off-by: Zhao Chao <zhaochao1984@gmail.com>
---
 src/ovirt_hosted_engine_setup/constants.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/ovirt_hosted_engine_setup/constants.py b/src/ovirt_hosted_engine_setup/constants.py
index 63c0946..34988f6 100644
--- a/src/ovirt_hosted_engine_setup/constants.py
+++ b/src/ovirt_hosted_engine_setup/constants.py
@@ -392,7 +392,6 @@ class EngineEnv(object):
 
     @ohostedattrs(
         summary=True,
-        answerfile=True,
         description=_('Host name for web application'),
     )
     def APP_HOST_NAME(self):
-- 
2.1.3

